lib/authorization.rb

```diff
@@ -55,8 +55,10 @@ def self.generate_default_acls(subject, object)
       acls: [
         {subjects: [subject], actions: ["Project:view", "Project:delete", "Project:user", "Project:policy", "Project:billing", "Project:github"], objects: [object]},
         {subjects: [subject], actions: ["Vm:view", "Vm:create", "Vm:delete"], objects: [object]},
+        {subjects: [subject], actions: ["Vm:Firewall:view", "Vm:Firewall:edit"], objects: [object]},
```

Comment:
How about create/delete instead of edit like we do in other resources?

Reply:
Yes for consistency but no for the use of it. For firewalls, it's the same thing to have delete vs create access. I can simply invalidate all of the firewalls simply adding `0.0.0.0/0` an `::/0`.